internal/common/errors.go

```diff
@@ -0,0 +1,51 @@
+package common
+
+import (
+	"errors"
+	"github.com/gofiber/fiber/v2"
+	"gorm.io/gorm"
+	"log"
+)
+
+func ServerError(ctx *fiber.Ctx, err error) error {
+	log.Print(err.Error())
```

Comment:
We should use Fibers' logger, but I'm not sure we want to explicitly log here

Reply:
Thanks, i will also correct all the errors trigger from the CI